### PR TITLE
Feature/event expiration

### PR DESCRIPTION
**Pull request by jwebbed**

### CODE REVIEW

File: events.html [hunk 1]
```diff
@@ -3,17 +3,20 @@
...
-      <a class="post-link" href="{{ post.url | prepend: site.baseurl }}">{{ post.title }}</a>
-      {% if post.datestart and post.dateend %}
-        <p>{{ post.datestart | date: "%b %-d" }} - {{ post.dateend | date: "%b %-d" }}</p>
-      {% elsif post.date %}
-        <p>{{ post.date | date: "%b %-d" }}</p>
-      {% endif %}
-      <p>{{ post.content | strip_html | truncatewords: 50 }}</p>
-    </li>
+  {% assign expires = post.date | date: "%s" | plus: 86400 | date: "%s" %}
+    {% if expires > curr_time %}
```
Comment (elliottsj) [inline events.html:10]: Does `if post.date <= site.time` work? No need to add a day to the date

Comment (jwebbed) [inline events.html:10]: We can't just do a straight comparison, it wasn't compiling as mentioned in https://github.com/cssu/cssu.cdf.toronto.edu/commit/776f77199193be0bc9e0f348bab7448708de2137

Comment (arkon) [inline events.html:10]: I think he means just do `expires >= curr_time` and get rid of the `plus: 86400`.

Comment (jwebbed) [inline events.html:10]: I'll test it

Comment (jwebbed) [inline events.html:10]: That dosn't work, I just tested it

**Comment by elliottsj** [comment]
Closed via #6

**Pull request closed without merging**